internal/command/testdata/test/pass_with_run_block_variable/main.tf

```diff
@@ -0,0 +1,31 @@
+# main.tf
+
+variable "aws_access_key_id" {
+  type    = string
+  default = "unix:///Users/siddharthasonker/.docker/run/docker.sock"
+}
+
+output "aws_access_key_id" {
+  sensitive = false
+  value = var.aws_access_key_id
+}
+
+terraform {
+  required_providers {
+    docker = {
+      source  = "kreuzwerker/docker"
```

Comment:
can we simply use 'provider "test"' like other tests？

Reply:
I agree, using a real provider here is a little strange.